src/Pickles/Pickles/ObjectModel/Mapper.cs

```diff
@@ -190,21 +190,23 @@ public Location MapToLocation(G.Location location)
             return location != null ? new Location { Column = location.Column, Line = location.Line } : null;
         }
 
-        public Scenario MapToScenario(G.Scenario scenario)
+        public Scenario MapToScenario(G.Scenario scenario, string[] tagsToHide)
```

Comment:
It would be more elegant to make the tagsToHide parameter a params array (params string[] tagsToHide). That would remove the need to pass in null arguments on many calls of the method.